app/views/user_mailer/password_reset.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <p>Dear SSID user,</p>
+
+  <p>The password for your account (<%= @user_email %>) was successfully reset on <%= @reset_time %>.</p>
+
+  <p>If you have any questions, please <%= mail_to "wing.nus@gmail.com", "contact us" %></p>
+
+  <p>Thank you.</p>
+  
+  <p>The SSID Team</p>
```

Comment:
Use constants instead of strings? "The SSID Team" => SSID_Maintainer

Reply:
Same as above.